packages/ui-core/src/components/card.component.tsx

```diff
@@ -18,7 +19,7 @@ const Card = styled.div<Props>`
 
 Card.defaultProps = {
   border: 'none',
-  backgroundColor: colors.white,
+  backgroundColor: '#fff',
```

Comment:
is it necessary? let's handle it with `@keen/colors`

Reply:
Thanks missed it during extracting <Card /> to ui-core 👍